docs/website/content/2.configuration/4.block/backend.md

```diff
@@ -226,3 +226,10 @@ If an attribute is set in both the _referenced_ and the _refining_ block, the va
 * `disable_connection_reuse`,
 * `http2` and
 * `max_connections`.
+
+::blocks
+---
+values: null
```

Comment:
Seems not to work for all blocks?

Additionally this would map to the Vue Component : `website/components/content/Blocks.vue` which does not exist. So no rendering takes place.

Reply:
We should skip writing if values is nil.

Reply:
> Seems not to work for all blocks?

Not all blocks have child blocks with `"docs"`. What would be the expected value?

> Additionally this would map to the Vue Component : `website/components/content/Blocks.vue` which does not exist.

Not _yet_.